src/main/java/RacingGame.java

```diff
@@ -1,32 +1,34 @@
 import domain.Car;
-import view.ResultView;
 
-import java.util.ArrayList;
-import java.util.List;
-import java.util.Random;
+import java.util.*;
+import java.util.stream.Collectors;
 
 public class RacingGame {
 
-    private static final Integer CONDITION = 4;
-    Random random = new Random();
+    private static final Integer CONDITON = 4;
+    private Random random = new Random();
 
-    public List<Car> makeCars(int carCnt) {
-        List<Car> cars = new ArrayList<Car>(carCnt);
-        for (int i = 0; i < carCnt; i++) {
-            cars.add(new Car());
-        }
-        return cars;
+
+    public List<Car> makeCars(String[] carNames) {
+        return Arrays.stream(carNames)
+                .map(name -> new Car(name))
+                .collect(Collectors.toList());
     }
 
     public void game(List<Car> cars) {
         cars.forEach(car -> {
-            race(car, random.nextInt(10));
+            car.moveWithConditon(random.nextInt(10), CONDITON);
         });
     }
 
-    public void race(Car car, int value) {
-        if (value >= CONDITION) {
-            car.move();
-        }
+    public List<Car> getWinners(List<Car> cars) {
+        int max = cars.stream()
+                .mapToInt(Car::getDistance)
+                .max()
+                .getAsInt();
+
+        return cars.stream().filter(
+                car -> max == car.getDistance()
+        ).collect(Collectors.toList());
```

Comment:
스트림을 1줄에 `.`를 1개만 작성하면 가독성에 좋습니다.
```suggestion
        return cars.stream()
                .filter(car -> max == car.getDistance())
                .collect(Collectors.toList());
```

그리고 car에서 distance값을 꺼내서 처리하지않고 max를 car객체로 넘겨서 처리해볼 수 있을까요?
이 [사이트](https://tecoble.techcourse.co.kr/post/2020-04-28-ask-instead-of-getter/)를 참고해보면 좋을 것 같아요.